src/bitcoin-cli.cpp

```diff
@@ -456,6 +458,15 @@ class NetinfoRequestHandler : public BaseRequestHandler
         if (conn_type == "addr-fetch") return "addr";
         return "";
     }
+    std::string FormatServices(const UniValue& services)
+    {
+        std::string str;
+        for (size_t i = 0; i < services.size(); ++i) {
+            const std::string s{services[i].get_str()};
```

Comment:
Can `NODE_NONE` ever be a value here?

https://github.com/bitcoin/bitcoin/blob/master/src/protocol.h#L312

Reply:
I think so. Using this, I see empty service names while a peer connection is setting up, and often from one or two (usually inbound tor) peers. It looks like `NODE_NONE` would be returned as an empty string by `serviceFlagsToStr()`.

Reply:
Thanks for clarifying, I was wondering whether the below line would cause `NODE_NONE` to show up as `n` but that should not be the case.